packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1703,7 +1703,10 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
               return widget.onGenerateInitialRoutes!(initialRouteName);
             },
           onUnknownRoute: _onUnknownRoute,
-          observers: widget.navigatorObservers!,
+          observers: <NavigatorObserver>[
+            TapRegionNavigatorObserver(),
```

Comment:
this is not good, if may broke for nested navigator. I think we may need to a way to mute TapRegion underneath the active route without using navigator observer. I can think of two ways.
1. TapRegion awares of ModalRoute.isActive and register/unregister itself
2. A new widget to hide TapRegionRegistry in the subtree, and then ModalRoute or Overlay use that to hide TapRegionRegistry for not opaque